### PR TITLE
816 deploy fink broker on fink integration platform

### LINKED ISSUE #816: Deploy fink-broker on fink integration platform
**Describe the issue**

Fink-broker will be deployed on fink integration platform (i.e. openstack magnum)

**Expected behaviour**

Fink should retrieve an process ZTF data

### PR DESCRIPTION
**Pull request by fjammes**
**IMPORTANT: Please create an issue first before opening a Pull Request.**
Linked to issue(s): Closes #816 

## What changes were proposed in this pull request?

Improvments required to install fink in production on k8s + documentation.

How is the issue this PR is referenced against solved with this PR?

fink is running on integration platform, it still require some polishing which will be performed in a forthcoming ticket.

## How was this patch tested?

In CI platform

### CODE REVIEW

File: doc/prod.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+```shell
+ln -s fink-broker/deploy/finkctl.yaml finkctl.yaml
+cat finkctl.secret.yaml
+s3:
+  id: "S3_USER_ID"
+  secret: "S3_SECRET"
+distribution:
+  kafka:
+    username: "spark"
```
Comment (JulienPeloton) [inline doc/prod.md:53]: Can you put placeholder here!

File: doc/prod.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    # If empty, password is set to "kubectl get -n kafka secrets/fink-producer --template={{.data.password}} | base64 --decode"
+    # this is used for integration tests and CI which use a local kafka cluster
+    password: "spark-secret"
```
Comment (JulienPeloton) [inline doc/prod.md:56]: Can you put placeholder here!

**Review by JulienPeloton — APPROVED** [review]
Thank you @fjammes ! I left some comments, but then you can merge :-)

File: .github/workflows/e2e-science.yml [hunk 3]
```diff
@@ -4,12 +4,13 @@ on:
...
-  # ciux does not support pull_request when trying not to build image
-  #   branches:
-  #     - master
-  # pull_request:
-  #   branches:
-  #     - master
+  # TODO ciux does not support pull_request when trying not to build image
+    branches:
+      - master
+      - 816-deploy-fink-broker-on-fink-integration-platform
```
[the remark this answers is not in the document]
Comment (fjammes) [reply .github/workflows/e2e-science.yml:10]: :+1:

File: doc/prod.md (hunk 1 above)
Comment (fjammes) [reply doc/prod.md:53]: :+1:

File: doc/prod.md (hunk 2 above)
Comment (fjammes) [reply doc/prod.md:56]: :+1:

**Pull request merged**